dds/idl/langmap_generator.cpp

```diff
@@ -388,16 +388,16 @@ class GeneratorBase
     const Classification cls = classify(actual_field_type);
     if (cls & (CL_PRIMITIVE | CL_ENUM)) {
       ss <<
-        "      this->_u." << name << " = other._u." << name << ";\n";
+        "    this->_u." << name << " = other._u." << name << ";\n";
     } else if (cls & CL_STRING) {
       ss <<
-        "      this->_u." << name << " = (other._u." << name << ") ? ::CORBA::string_dup(other._u." << name << ") : 0 ;\n";
+        "    this->_u." << name << " = (other._u." << name << ") ? ::CORBA::string_dup(other._u." << name << ") : 0 ;\n";
     } else if (cls & CL_ARRAY) {
       ss <<
-        "    this->_u." << name << " = (other._u." << name << ") ? " << map_type(field_type) << "_dup(other._u." << name << ") : 0 ;\n";
+        "  this->_u." << name << " = (other._u." << name << ") ? " << map_type(field_type) << "_dup(other._u." << name << ") : 0 ;\n";
```

Comment:
Does the array get indented less than the others, or is this a typo?

Reply:
Probably a bug, looks like we need coverage for the case where an array is directly in a union.  I'll add...

Reply:
It's covered in tests/DCPS/Compiler/FooDef.idl (BigUnion).  I'll push a change.